README.md

```diff
@@ -295,7 +296,7 @@ well.
   be verified and whose static and `prototype` functions have all been replaced
   with test double functions using the same
   [imitation](https://github.com/testdouble/testdouble.js/blob/master/src/imitate/index.js)
-  mechanism described above
+  mechanism as described in `td.object(realObject)`
```

Comment:
"described above" is kind of unclear about where it's talking about. I updated it with what I inferred its reference to be, but I'm not sure I got this right.